README.md

```diff
@@ -1,20 +1,149 @@
-# OSmOSE_package
+# OSmOSE-toolkit
 
 
-## Installation
+## Presentation
 
-OSmOSE is currently in development, and no release exist yet. Thus, installing the package means using an unstable, unfinished version.
+OSmOSE toolkit is an open source suite of tools dedicated to the management and analysis of passive underwater acoustic data. Among other key features, our toolkit has been adapted to be deployed on a cluster infrastructure; below you will find installation procedures using the (DATARMOR)[https://www.ifremer.fr/fr/infrastructures-de-recherche/le-supercalculateur-datarmor] cluster of IFREMER, on which our production version runs.
+
+## For users
+
+### Quick start  in local
+
+Users will be interested in the stable versions of our toolkit, which are released on a regular basis and are available (here)[https://github.com/Project-OSmOSE/osmose-toolkit/releases/tag/v0.1.0]. We discourage users to install the toolkit from our git repository, as it is under permanent development and thus potentially unstable and unfinished.
+
+### Quick start on DATARMOR
+
+The newest stable version can be directly used within a Jupyter Hub session of Datarmor by setting the kernel to `osmose`. A set of "getting started" Jupyter notebooks are available from our datawork `/home/datawork-osmose/osmose-datarmor`, which covers basic use cases from your dataset uploading to first AI model development. To access the Datarmor cluster, you will need a user account, which may be provided upon request (please contact dorian.cazau@ensta-bretagne.fr).
+ 
+
+
+## For developers
+
+### Contribution workflow
```

Comment:
Contribution guidelines are typically at the bottom of the README or in the CONTRIBUTING.md file. Accepting this for now but this section will probably move eventually.